src/org/traccar/protocol/MxtProtocolDecoder.java

```diff
@@ -145,7 +145,7 @@ protected Object decode(
             }
 
             if (BitUtil.check(infoGroups, 4)) {
-                position.set(Position.KEY_HOURS, buf.readUnsignedInt());
+                position.set(Position.KEY_HOURS, buf.readUnsignedInt() * 60000);
```

Comment:
Sorry, haven't noticed it before. I think all of the conversions should be using `UnitsConverter`.